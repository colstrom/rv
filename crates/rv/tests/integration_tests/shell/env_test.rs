use crate::common::RvTest;
use insta::assert_snapshot;

#[test]
fn test_shell_env_succeeds() {
    let test = RvTest::new();
    let output = test.rv(&["shell", "env"]);

    assert_snapshot!(output.normalized_stdout());
    assert!(output.success());
}
