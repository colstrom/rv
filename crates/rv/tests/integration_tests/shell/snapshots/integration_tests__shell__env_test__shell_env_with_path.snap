---
source: crates/rv/tests/integration_tests/shell/env_test.rs
expression: output.normalized_stdout()
---
unset RUBY_ROOT RUBY_ENGINE RUBY_VERSION RUBYOPT GEM_ROOT GEM_HOME GEM_PATH
export PATH=/tmp/bin
hash -r
